.gitlab-ci.yml

```diff
@@ -43,7 +43,7 @@ builder-image:
   before_script:
     - echo HOST=$HOST
     - |
-      if [ "$HOST" = "x86_64-apple-darwin16" ]; then
+      if [ "$HOST" = "x86_64-apple-darwin18" ]; then
```

Comment:
imo 20419 shouldn't be marked partial

Reply:
https://github.com/dashpay/dash/pull/4378/commits/d451833380e9d11ff426102c61c3a90beabcbfa0